body.tex

```diff
@@ -0,0 +1,440 @@
+
+Patch release requests should use an RFC Jira issue which must be approved by the DMCCB.
+The issue should specify what needs to be fixed in the release and why, it shall specify DM issues that are requested to be included in the patch.
+
+This process is documented in \citeds{LDM-294}, sections 3.6 and 7.4.
+
+
+\subsection{Issue Management} \label{sec:issues}
+
+The release is identified in Jira using a release issue.
+
+All issues to be included in a release shall be added as blocking to the release issue.
+
+Note that by introducing the field \textit{Fix in Version(s)}, will permit having the same information in a much simpler way.
+Both release issue and \textit{Fix in Version(s)} fields can be used in parallel.
+Once the release is done, it will also be possible to complete the field \textit{Fix in Version(s)} with the exact release,
+ensuring in this way that each issue is fully documented and self consistent\footnote{process to be automated}.
+
+
+\subsection{Release Note} \label{sec:relnote}
+
+The release note documents the content of a release.
+
+The following information should be  provided:
+\begin{itemize}
+\item Installation instructions. This is usually a manually written set of instructions.
+\item Narrative section describing the content of the release, this summary should be written by a person not auto generated.
+The T/CAM is responsible for making sure this is provided.
+\item List of jira issues included in a release. This information can be extracted from Github. Completed epics will be highlighted while other issues will  listed to be comprehensive.
+\item Technical information like the GitHub tag, dependencies,  etc. can be extracted automatically from Github or other tools.
+\end{itemize}
+
+
+\newpage
+\section{Software Release Procedure} \label{sec:releaseprocedure}
+
+This release procedure has been derived from the \textit{Stack release playbook} \citeds{SQR-016}.
+
+\subsection{Development} \label{sec:dev}
+
+Development activities are not part of the release process, but are the starting point for a stable and reliable master branch in all Github software packages.
+The Developer Guide lays out the guidelines and process. All changes are done on ticket branches and reviewed using the Pull Request mechanism before merging to master.
+
+Each time a change is merged into master, the following activities should be performed:
+
+\begin{itemize}
+\item continuous integration build of the Github software package (SW product)
+\item if unit tests pass, generate binary packages
+\item build downstream dependencies: CI build on SW products that depend from the newly build SW product.
+\end{itemize}
+
+At the moment, continuous integration is done on a ticket branch prior to merging changes to master. Binary packages (EUPS) and docker images for distribution are made available with the nightly and weekly builds (see next section \ref{sec:weekly}).
```

Comment:
To be clear, is the above suggesting that Jenkins should _no longer_ run on ticket branches, or that it should run on _both_ ticket branches and on `master`? If the former, that sounds like a problem: developers should be able to test their changes before they merge them

Reply:
So far I am not proposing to change Jenkins approach in that respect.